debian/ubuntu-advantage-tools.postrm

```diff
@@ -19,9 +19,7 @@ remove_logs(){
 }
 
 remove_gpg_files(){
-    rm -f /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-infra-trusty.gpg
-    rm -f /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-apps.gpg
-    rm -f /etc/apt/trusted.gpg.d/ubuntu-advantage-fips.gpg
+    rm -f /etc/apt/trusted.gpg.d/ubuntu-advantage-*.gpg
```

Comment:
The only problem here is removing files we should not. Given that, I think that no other APT repository should have `ubuntu-advantage` as a prefix for the keys, and that it is safe and reasonable for us to remove it like this.

Reply:
I agree with that approach, but let's just confirm with @blackboxsw to see if this is really safe